src/events/processor/find-features/index.js

```
@@ -217,7 +217,7 @@ const generateFeatures = ({ locations, report, options, sourceRatings }) => {
       const clId = countryLevels.getIdFromLocation(location);
       if (clId) {
         const feature = await countryLevels.getFeature(clId);
-        storeFeature(feature, location);
+        if (feature !== null) storeFeature(feature, location);
```

We don't have to care for this, it should throw an exception.